server/src/test/java/keywhiz/KeywhizTestRunner.java

```diff
@@ -49,35 +40,7 @@ public KeywhizTestRunner(Class<?> klass) throws InitializationError {
     BootstrapLogging.bootstrap();
   }
 
-  private static final Injector injector = createInjector();
-
-  static Injector createInjector() {
-    KeywhizService service = new KeywhizService();
-    Bootstrap<KeywhizConfig> bootstrap = new Bootstrap<>(service);
-    service.initialize(bootstrap);
-
-    File yamlFile = new File(Resources.getResource("keywhiz-test.yaml").getFile());
-    Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
-    ObjectMapper objectMapper = bootstrap.getObjectMapper().copy();
-    KeywhizConfig config;
-    try {
-      config = new YamlConfigurationFactory<>(KeywhizConfig.class, validator, objectMapper, "dw")
-          .build(yamlFile);
-    } catch (IOException | ConfigurationException e) {
-      throw Throwables.propagate(e);
-    }
-
-    Environment environment = new Environment(service.getName(),
-        objectMapper,
-        validator,
-        bootstrap.getMetricRegistry(),
-        bootstrap.getClassLoader());
-
-    Injector injector = Guice.createInjector(new ServiceModule(config, environment));
-
-    service.setInjector(injector);
-    return injector;
-  }
```

Comment:
This logic is now stored in a dedicated class since it's used in multiple places now and is no longer test runner-specific.